examples/looker_embed_generator/looker_embed_generator.java

```diff
@@ -6,20 +6,38 @@
  * 
  *Requirements   : You must set the following environment variable
  *               : to establish credentials for the embed user:
+ *               : 
+ *               : This project also requires the use of google/GSON,
+ *               : a Java serialisation/deserialisation library. The
+ *               : github repo can be found here: 
+ *               :   https://github.com/google/gson
+ *               : 
  * 
  *               : export LOOKERKEY=<<Looker embed key>>   ## bash
  *               : set LOOKERKEY=<<Looker embed key>>      :: cmd
  *               : $env:LOOKERKEY = "<<Looker embed key>>" ## powershell 
  * 
- *Usage          : java looker_embed_generator <<environment>> <<embed_url>> [<<attribute>> <<filter>>]
+ *Usage          : java looker_embed_generator <<environment>> -e <<embed_url>> -u [<<attribute>> <<filter>>]
+ *               :
+ *               : eg: java looker_embed_generator.java prod dashboards/18
+ *               :     java looker_embed_generator.java prod looks/98 browser Chrome
```

Comment:
Again, needs a `-classpath` set with the gson JAR requirement.

Reply:
Ive updated the example commands.